crates/db_views/community/src/api.rs

```diff
@@ -317,7 +317,8 @@ pub struct ListMultiCommunitiesResponse {
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
 #[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
 pub struct GetMultiCommunity {
-  pub id: MultiCommunityId,
+  pub id: Option<MultiCommunityId>,
+  pub name: Option<String>,
```

Comment:
Add a TODO in here, or create an issue and reference this, that we should probably be using typed enums for these cases, when one is required. 

I'm not exactly sure how it would work with query params, but you or someone would know better.